pdal/PointView.cpp

```diff
@@ -100,26 +101,13 @@ void PointView::setFieldInternal(Dimension::Id dim, PointId idx,
 
 void PointView::calculateBounds(BOX2D& output) const
 {
-    for (PointId idx = 0; idx < size(); idx++)
-    {
-        double x = getFieldAs<double>(Dimension::Id::X, idx);
-        double y = getFieldAs<double>(Dimension::Id::Y, idx);
-
-        output.grow(x, y);
-    }
+    pdal::calculateBounds(const_cast<PointView&>(*this), output);
```

Comment:
@abellgithub Is this what you'd envisioned in terms of a redirect?

Reply:
Exactly.